Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -691,6 +697,11 @@ public void ReadXmlPrototype(XmlReader readerParent)
                     reader.Read();
                     Height = reader.ReadContentAsInt();
                     break;
+                case "Health":
+                    reader.Read();
+                    health.InitialHealth(reader.ReadContentAsFloat());
+                    health.MaxHealth = reader.ReadContentAsFloat();
```

Comment:
I've tried that. It silently breaks. The selection display doesn't show health at all.